apis/core/lib/domain/table/create.ts

```diff
@@ -0,0 +1,45 @@
+import { GraphPointer } from 'clownface'
+import { schema, rdf } from '@tpluscode/rdf-ns-builders'
+import { cc } from '@cube-creator/core/namespace'
+import { ResourceStore } from '../../ResourceStore'
+import * as id from '../identifiers'
+import { resourceStore } from '../resources'
+import { NamedNode } from 'rdf-js'
+
+interface CreateTableCommand {
+  tableCollection: GraphPointer
+  resource: GraphPointer
+  store?: ResourceStore
+}
+
+export async function createTable({
+  tableCollection,
+  resource,
+  store = resourceStore(),
+}: CreateTableCommand): Promise<GraphPointer> {
+  const label = resource.out(schema.name)
+  if (!label?.term) {
+    throw new Error('schema:name missing from the payload')
+  }
+
+  const csvMapping = tableCollection.out(cc.csvMapping)
+  if (!csvMapping?.term) {
+    throw new Error('cc:csvMapping missing from the payload')
+  }
+
+  if (!tableCollection?.term) {
```

Comment:
It looks like I'm seeing an Open World™ 😄

Reply:
Yeah but this:
<img width="806" alt="Screenshot 2020-10-27 at 16 10 35" src="https://user-images.githubusercontent.com/2022803/97321329-089ee680-186f-11eb-8b07-d470bf2aca45.png">
 is so much more useful than this:
<img width="1095" alt="Screenshot 2020-10-27 at 16 10 56" src="https://user-images.githubusercontent.com/2022803/97321334-0b99d700-186f-11eb-8e8a-66fca882cfc1.png">

😄

Reply:
I wasn't complaining about your code. Half my code ends up looking the same: with `if` statements every 2 lines... I was just saying that it's a direct consequence of the "Open World" assumption.